packages/api/test/functional/technology-order.spec.js

```diff
@@ -0,0 +1,182 @@
+const { trait, test } = use('Test/Suite')('Technology Order');
+
+const Technology = use('App/Models/Technology');
+const User = use('App/Models/User');
+const TechnologyOrder = use('App/Models/TechnologyOrder');
+
+trait('Test/ApiClient');
+trait('Auth/Client');
+trait('DatabaseTransactions');
+
+const {
+	antl,
+	errors,
+	errorPayload,
+	fundingStatuses,
+	technologyUseStatuses,
+	orderStatuses,
+} = require('../../app/Utils');
```

Comment:
É besteira, mas, penso que seria mais "sintático" usar os imports e configs desta forma:
```suggestion
const { trait, test } = use('Test/Suite')('Technology Order');
const Technology = use('App/Models/Technology');
const User = use('App/Models/User');
const TechnologyOrder = use('App/Models/TechnologyOrder');
const {
	antl,
	errors,
	errorPayload,
	fundingStatuses,
	technologyUseStatuses,
	orderStatuses,
} = require('../../app/Utils');

trait('Test/ApiClient');
trait('Auth/Client');
trait('DatabaseTransactions');
```